lua/core/utils/init.lua

```diff
@@ -258,5 +258,6 @@ function astronvim.cmd(cmd, show_error)
 end
 
 require "core.utils.updater"
+require "core.utils.ux"
```

Comment:
could we rename this to `core.utils.ui` to match the "UI" menu option?

Reply:
I thought about the same name but I didn't do so due to existing ui.lua file.
Now that I see you removed ui.lua, 
I will rename.
I will also drop tailing "s" from "Toggle URL highlights" to be consistent.